Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -11,20 +11,27 @@ protocol Validation {
 }
 
 class SignInViewController: UIViewController, Validation {
+    // MARK: - IBOulets
     @IBOutlet weak private var txtUsername: UITextField!
```

Comment:
In your variables name, could you use full words instead of abbreviations or shortened words just to make them more clear

Reply:
Will do so. Thank you for the suggestion 👍🏻

Reply:
Done